src/hamiltonian/Hamiltonian_Model/Model_double_well.h

```diff
@@ -1,5 +1,5 @@
 /*********************************************************************************
-* Copyright (C) 2015-2017 Brendan A. Smith , Alexey V. Akimov 
+* Copyright (C) 2015-2017 Alexey V. Akimov

```

Comment:
yes, add your name, please, but in a separate line (on top of the existing one), the new line should read like:
Copyright (C) 2018 Brendan A. Smith , Alexey V. Akimov